File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int j = 0;
        for(int num : nums) {
            if(j < 1 || nums[j - 1] != num) 
                nums[j++] = num;
        }
        return j;
    }
}

File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        l, cnt, n = 1, 1, len(nums)
        for i in range(1, n):
            if nums[i] != nums[i - 1]:
                nums[l] = nums[i]
                l += 1
        return l
            